internal/databases/mysql/backup_push_handler.go

```diff
@@ -59,7 +59,7 @@ func HandleBackupPush(uploader *internal.Uploader, backupCmd *exec.Cmd, isPerman
 		CompressedSize:   uploadedSize,
 		UncompressedSize: rawSize,
 		IsPermanent:      isPermanent,
-		UserData:         userData,
+		UserData:         internal.UnmarshalSentinelUserData(userData),
```

Comment:
Any error hanlding during unmarshalling ?

Reply:
it is already being handled in `internal.UnmarshalSentinelUserData`. If failed to parse as JSON, it will leave the string as it is.